src/Util/Todo.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Sonata\AdminBundle\Util;
+
+use Doctrine\Common\Util\ClassUtils;
+use Doctrine\Inflector\InflectorFactory;
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+
+class Todo
```

Comment:
Any Idea about the name of this service @sonata-project/contributors ? @wbloszyk ?

Reply:
MethodsHelper ??